cmd/app/app.go

```diff
@@ -77,6 +81,21 @@ func (a *App) Run() error {
 	return ui.StartUILoop()
 }
 
+func (a *App) setMaxProcs() {
+	maxProcs := a.Flags.MaxCores
+
+	// if 'maxcores' value is zero or below zero, we repeat the logic of GOMAXPROCS(n) when n < 1
+	if maxProcs > runtime.NumCPU() || maxProcs <= 0 {
+		maxProcs = runtime.NumCPU()
```

Comment:
This line has no effect.

Reply:
Changed logic:
If user set maxprocs below 1 we don't care and use default setting